src/FormBuilder/resources/js/form-builder/src/supports/field-settings/index.tsx

```diff
@@ -0,0 +1,20 @@
+import {addFilter} from "@wordpress/hooks";
+
+import FieldSettingsHOC from "./FieldSettingsHOC";
+import updateBlockTypes from "./updateBlockTypes";
+import {DisplaySettingsFill, FieldSettingsFill} from "./slots";
+
+/**
+ * Registers the necessary hooks for supports the field settings
+ *
+ * @unreleased
+ */
+export default function registerHooks() {
+    addFilter('editor.BlockEdit', 'givewp/supports/field-settings-hoc', FieldSettingsHOC);
+    addFilter('blocks.registerBlockType', 'givewp/supports/field-settings-attributes', updateBlockTypes);
+
+    // Mounts the field settings inspector slots so other plugins can extend the inspector controls
+    window.givewp.slots = window.givewp.slots || {};
+    window.givewp.slots.FieldSettingsFill = FieldSettingsFill;
+    window.givewp.slots.DisplaySettingsFill = DisplaySettingsFill;
```

Comment:
I wasn't sure which TS Window instance to update to document these additions to the window.

Reply:
@JasonTheAdams actually can we change this to `window.givewp.form.slots`? That way its scoped to our form object along with `window.givewp.form.blocks`.  

Also, we are not isolated from the admin scripts so other admin defined objects like the legacy `Give` is defined on the window. I imagine we'll do something similar for the detail pages like `window.givewp.donation.data`, `window.givewp.donation.slots` so just want to make sure we scope our window objects to avoid any conflicts.

Reply:
Got it. Resolved in 3c3728a